README.md

```diff
@@ -141,7 +141,10 @@ $ preact watch
 
     --src           Specify source directory  (default src)
     --cwd           A directory to use instead of $PWD  (default .)
-    --esm           Builds ES-2015 bundles for your code.  (default true)
+    --devServer     Determine if dev server should be enabled  (default true)
+    --dest          Specify output directory if dev server is disabled  (default build)
+    --ignore        Path relative to src to be ignored during watch if dev server is disabled
+	  --esm           Builds ES-2015 bundles for your code.  (default true)
```

Comment:
nitpick: The slightly different indentation here is quite noticeable on the rendered file